src/config/Config.zig

```diff
@@ -3381,6 +3381,14 @@ fn loadTheme(self: *Config, theme: Theme) !void {
         name,
         &self._diagnostics,
     )) orelse return;
+    const stat = try themefile.file.stat();
+    switch (stat.kind) {
+        .file => {},
+        else => {
+            log.warn("not loading file \"{s}\" because it is a: {s}", .{ themefile.path, @tagName(stat.kind) });
```

Comment:
Useless nitpick: "it is" here but "it's" in `theme.zig`.
```suggestion
            log.warn("not loading file \"{s}\" because it's a: {s}", .{ themefile.path, @tagName(stat.kind) });
```

Reply:
Actually, i just removed the stat test from this location. Going to rely on `themepkg.open` never returning anything but a file.